src/main/java/Storage.java

```diff
@@ -0,0 +1,74 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath;
+    public Storage(String file) {
+        this.filePath = file;
+        this.checkDirectory();
+        this.createFile();
+    }
+    public void checkDirectory() {
+        String PATH = "./data";
+        File directory = new File(PATH);
+        if (!directory.exists()) {
+            directory.mkdir();
+        }
+    }
+    public  void createFile() {
+        try {
+            File myObj = new File(this.filePath);
+            if (myObj.createNewFile()) {
+                System.out.println("File created: " + myObj.getName());
+            }
+        } catch (IOException e) {
+            System.out.println("An error has occured");
+        }
+    }
+    public ArrayList<Task> load() throws DukeException {
+        ArrayList<Task> tasks = new ArrayList<>();
+        try {
+            File myObj = new File(this.filePath);
+            Scanner myReader = new Scanner(myObj);
+            while (myReader.hasNextLine()) {
+                String data = myReader.nextLine();
+                String[] taskBreakdown = data.split("\\| ");
+                String taskType = taskBreakdown[0].trim();
+                boolean isTaskCompleted = taskBreakdown[1].trim().equals("1");
+                String taskMessage = taskBreakdown[2].trim();
+                String taskDate = taskBreakdown.length > 3 ? taskBreakdown[3].trim(): "";
+
+                if (taskType.equals("T")) {
+                    tasks.add(new Todo("todo " + taskMessage, isTaskCompleted));
+                }
+                else if (taskType.equals("E")) {
```

Comment:
perhaps to you could start the else clause to the curly bracket to meet the coding standard?
```
if (condition) {
    statements;
} else if (condition) {
    statements;
} else {
    statements;
}
```